src/unix/fs.c

```diff
@@ -231,7 +231,29 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 
 
 static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
-  return mkdtemp((char*) req->path) ? 0 : -1;
+  char *path = (char*) req->path;
+
+#if defined(__MVS__)
+  /* There is no mkdtemp. So instead use mktemp to generate a
```

Comment:
this makes me a bit uneasy :-S we do have a hand-rolled mkdtemp implementation on Windows. Alas it won't be easy to extract because of the wchat conversions.

Maybe we can add it here again? (it was taken from OpenBSD)

@libuv/collaborators thoughts?